### PR TITLE
Allow clock pins to be distinguishable

### PR DESCRIPTION
**Pull request by hanwu85**
Addressing issue  #759

### CODE REVIEW

**Comment by xsrvmy** [comment]
The sentence is not well formed:
distinguishable pins -> pins distinguishable
is permitted -> are permitted

Personally I think just changing "must not be" to "may be" in the original wording if clearer.

**Review by Ivan-Brigidano — CHANGES_REQUESTED** [review]
@hanwu85 I don’t think this change should be done until the Committee decide to change or not the Guideline

**Comment by hanwu85** [comment]
Yeah, initially I accidentally click the merge to draft instead of a PR.  I put this PR for review, if it is not passed we can always revers that change.

File: wca-guidelines.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ To be more informative, each Guideline is classified using one of the following
 - 3h++) [EXAMPLE] Examples of enhancements include: new moves are possible, normal moves are impossible, more pieces or faces are visible, colors on the backside of the puzzle are visible, moves are done automatically, or the puzzles have more/different solved states.
 - 3h2++) [CLARIFICATION] "Stickerless" puzzles that significantly differ from most mass-produced "stickerless" puzzles are only permitted at the discretion of the WCA Delegate.
 - 3j+) [REMINDER] In the past, engraved/embossed parts have been permitted. This is no longer permitted.
-- 3j++) [CLARIFICATION] On Clock, pins must not be distinguishable from any other pin of the same side.
+- 3j++) [CLARIFICATION] Clocks with pins distinguishable from any other pin of the same side is permitted.
```
Comment (Ivan-Brigidano) [inline wca-guidelines.md]:
```suggestion
- 3j++) [CLARIFICATION] On Clock, pins may be distinguishable from any other pin of the same side.
```

**Comment by Ivan-Brigidano** [comment]
(Closing until 2020 Regulations are released. Will reopen if 2020 Regulations include it)

File: wca-guidelines.md (hunk 1 above)
Comment (Ivan-Brigidano) [reply wca-guidelines.md]: @hanwu85 don't you think this style would be easier to read and would match 3j2+?

Comment (hanwu85) [reply wca-guidelines.md]: My brain was stuck and did not come up with this simple one...Thanks!

**Review by Ivan-Brigidano — APPROVED** [review]

**Comment by lgarron** [comment]
This change was fairly uncontroversial. Merging!

**Pull request merged**